metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/WorkflowExecutionFactory.java

```diff
@@ -247,6 +258,17 @@ private void setupXsltIdForPluginMetadata(Dataset dataset,
     pluginMetadata.setLanguage(dataset.getLanguage().name().toLowerCase(Locale.US));
   }
 
+  private void setupDepublishPluginMetadata(Dataset dataset,
```

Comment:
I guess at this point we have no certainty that the depublish plugin is the only plugin of any kind in the workflow. Should the WorkflowUtils, when validating the workflow, not check that a workflow that has a depublication cannot have anything else?

Reply:
Actually there is a check that validates what you just mentioned.

Reply:
Oh, I missed it. Ok then.